packages/common/src/audius-query/createApi.ts

```diff
@@ -132,7 +126,7 @@ const addEndpointToSlice = <NormalizedData>(
     ) => {
       const { fetchArgs } = action.payload
       const key = getKeyFromFetchArgs(fetchArgs)
-      const scopedState = { ...state[endpointName][key] } ?? initState
+      const scopedState = { ...state[endpointName][key] }
```

Comment:
I think we can do `state[endpointName][key] ? {...state[endpointName][key]} : initState`

Reply:
Awesome, will update!

Reply:
or maybe even `{ ...initState, ...state[endpointName][key] }`